internal/controller/components/dashboard/dashboard_controller_params_test.go

```diff
@@ -0,0 +1,496 @@
+// This file contains tests for dashboard controller parameter functionality.
+// These tests verify the dashboardctrl.SetKustomizedParams function and related parameter logic.
+package dashboard_test
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"github.com/opendatahub-io/opendatahub-operator/v2/api/common"
+	componentApi "github.com/opendatahub-io/opendatahub-operator/v2/api/components/v1alpha1"
+	dsciv1 "github.com/opendatahub-io/opendatahub-operator/v2/api/dscinitialization/v1"
+	dashboardctrl "github.com/opendatahub-io/opendatahub-operator/v2/internal/controller/components/dashboard"
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/cluster"
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/cluster/gvk"
+	odhtypes "github.com/opendatahub-io/opendatahub-operator/v2/pkg/controller/types"
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/utils/test/fakeclient"
+
+	. "github.com/onsi/gomega"
+)
+
+const paramsEnvFileName = "params.env"
+const errorNoManifestsAvailable = "no manifests available"
+
+func TestSetKustomizedParams(t *testing.T) {
```

Comment:
Those tests are not equivalent to the ones in `internal/controller/components/dashboard/dashboard_controller_kustomize_test.go`?

Reply:
yep, will consolidate both into one single test file